DPG/Tasks/AOTTrack/qaEventTrack.cxx

```diff
@@ -61,6 +61,9 @@ struct qaEventTrack {
   // options to select specific events
   Configurable<bool> selectGoodEvents{"selectGoodEvents", true, "select good events"};
 
+  // option to apply a timeframe cut
+  Configurable<bool> tfCut{"tfCut", true, "applies timeframe cut"};
```

Comment:
I would put this to `false` as default value, for backward compatibility. If we decide then to switch it on as default in our analyses, then we'll change the configuration